wiki/Gameplay/Daily_challenge/fr.md

```diff
@@ -0,0 +1,60 @@
+# Défi quotidien
+
+Le **défi quotidien** est un mode multijoueur de [osu!(lazer)](/wiki/Client/Release_stream/Lazer) où les joueurs peuvent accumuler une série en réussissant à passer des [beatmaps](/wiki/Beatmap) quotidiennement. La difficulté augmente de jour en jour et se réinitialise toutes les semaines.
+
+Chaque beatmap est soigneusement sélectionnée parmi la liste des [Featured Artists](/wiki/People/Featured_Artists), et parfois certaines maps imposent un mod aux joueurs, les obligeant à passer la map avec le mod de jeu sélectionné.
+
+## Menu du jeu
+
+Depuis la page principale du jeu, le menu des défis quotidiens est accessible en suivant les étapes suivantes:
+
+1. Cliquez sur l’icône `jouez` ou appuyez sur `P`.
+2. Cliquez sur l’icône `défi quotidien` ou appuyez sur `D`.
+
+![](img/daily-challenge-menu.png "Capture d'écran du menu du défi quotidien du 7 novembre 2024.")
+
+En entrant, il est présentée au joueur une introduction, montrant la beatmap à passer ainsi que les mods imposés.
+
+Après l'introduction, le joueur consulter les données relatives au défi du jour. Le côté gauche affiche diverses informations en rapport avec le [score](/wiki/Gameplay/Score), comme le nombre total de joueurs ayant réussi la beatmap et le score cumulé de tous les joueurs. Au centre, un classement montre les meilleurs scores obtenus par les joueurs. À droite, les joueurs peuvent discuter du défi quotidien dans un chat.
+
+## Plage de difficulté
+
+![](img/Beatmap-selection-criteria.png "Niveaux d'étoiles spécifiques que les beatmaps doivent respecter pour toute la semaine.")
+
+## Paliers de séries
+
+Les séries de parties sont affichés sur le profil du joueur avec différentes couleurs en fonction de certain paliers atteints:
+
+<!-- tier images: https://www.figma.com/design/tc79qAgJ35KQvdTO0Oj3dN/Daily-Challenge-Counter?node-id=0-1&t=xjRm9Ke0tUMtAQlh-1 -->
+
+|  | Niveau | Participation totale | Série quotidienne | Série hebdomadaire |
+| --: | :-: | :-: | :-: | :-: |
+| ![Brillant tier](img/tiers/lustrous.png) | Brillant | 1080 jours | 360 jours | 53 semaines |
+| ![Radiant tier](img/tiers/radiant.png) | Radiant | 720 jours | 240 jours | 36 semaines |
+| ![Rhodium tier](img/tiers/rhodium.png) | Rhodium | 360 jours | 120 jours | 19 semaines |
+| ![Platine tier](img/tiers/platinum.png) | Platine | 180 jours | 60 jours | 10 semaines |
+| ![Or tier](img/tiers/gold.png) | Or | 90 jours | 30 jours | 6 semaines |
+| ![Argent tier](img/tiers/silver.png) | Argent | 30 jours | 10 jours | 3 semaines |
+| ![Bronze tier](img/tiers/bronze.png) | Bronze | 15 jours | 5 jours | 2 semaines |
+| ![Fer tier](img/tiers/iron.png) | Fer | moins de 15 jours | moins de 5 jours | moins de 2 semaines |
+
+## Contributeurs
+
+Le projet est organisé par ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976). Les membres de la communauté suivants sont responsables du choix des beatmaps:
```

Comment:
```suggestion
Le projet est organisé par ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976). Les membres de la communauté suivants sont responsables du choix des beatmaps :
```

Reply:
In other versions, they don't put a space between a word and a :

Reply:
C'est censé être une erreur de grammaire.  Les deux points sont toujours précédés d'un espace insécable. C'est peut-être dû à un oubli parce qu'en anglais il n'y a pas cette règle typographique.

Reply:
I checked that, you are right. I will commit it when I can.